.gitignore

```diff
@@ -1,6 +1,8 @@
 /parts/
 /stage/
 /prime/
+.vscode/
```

Comment:
don't folks normally put this in their global gitignore?

https://blog.martinhujer.cz/dont-put-idea-vscode-directories-to-projects-gitignore/

I don't really know how i land on this...

Reply:
TIL global git ignore :smile: